### PR TITLE
Modified how the full screen rules work

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: src/app/public/modules/flyout/flyout.component.html [hunk 1]
```diff
@@ -22,6 +25,10 @@
     [attr.aria-valuenow]="flyoutWidth"
     [max]="config.maxWidth"
     [min]="config.minWidth"
+    [ngClass]="{
+      'sky-flyout-hidden': !isOpen && !isOpening,
+      'sky-flyout-fullscreen': isFullscreen
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/flyout/flyout.component.html]: `.sky-flyout-hidden` doesn't actually have any style declarations for this `.sky-flyout-resize-handle` element. Did you mean to put it here?

On second thought, do we really need these two lines at all? We could infer the input should take different styles in the parent `div` `ngClass` triggers. For example, you could rewrite your scss to look like this:
```
.sky-flyout {
 ...
  &.sky-flyout-fullscreen {
    min-width: 100%;
    max-width: 100%;

      .sky-flyout-resize-handle {
        // CSS for handle when fullscreen
      }
  }
  &.sky-flyout-hidden {
    visibility: hidden;

      .sky-flyout-resize-handle {
        // CSS for handle when hidden
      }
  }
...
```

File: src/app/public/modules/flyout/flyout.component.ts [hunk 2]
```diff
@@ -189,6 +191,12 @@ export class SkyFlyoutComponent implements OnDestroy, OnInit {
       this.updateBreakpointAndResponsiveClass(this.flyoutWidth);
     }
 
+    if ((window.innerWidth - this.windowBufferSize) < this.config.minWidth) {
+      this.isFullscreen = true;
+    } else {
+      this.isFullscreen = false;
+    }
+
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/flyout/flyout.component.ts]: Should we isolate this logic? I see it repeated down below... Something like `setFullscreen()` / `checkFullscreen()` etc...?

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/flyout/flyout.component.ts]: Done

File: src/app/public/modules/flyout/flyout.component.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/flyout/flyout.component.html]: Good call. Done

**Pull request merged**